image-trigger

```diff
@@ -66,13 +66,14 @@ REFRESH = {
     "rhel-9-4": {},
     "rhel4edge": {},
     "services": REFRESH_30,
+    "opensuse-tumbleweed": REFRESH_30,
+    "opensuse-microos": REFRESH_30,
 }
 
 
 def main():
-    parser = argparse.ArgumentParser(description='Ensure necessary issue exists for image refresh')
```

Comment:
Most of the changes you commited seems to be from some linter. Could you please drop those? (or at least separate commit but mind we have our own linter so they have to be compatible)